LayoutTests/platform/mac-wk2/TestExpectations

```diff
@@ -717,8 +720,8 @@ webkit.org/b/141085 http/tests/media/video-query-url.html [ Pass Timeout ]
 http/tests/inspector/network/local-resource-override-mapped-to-directory.html [ Pass ]
 http/tests/inspector/network/local-resource-override-mapped-to-file.html [ Pass ]
 
-# Network throttling is currently only enabled on Cocoa platforms.
-http/tests/inspector/network/setEmulatedConditions.html [ Pass ]
+# Network throttling is not supported in the unified HTTP stack.
```

Comment:
Network throttling is exposed as a feature by Web Inspector, so does this mean that WebKit cannot switch?

Reply:
This is not a feature enabled by default on customer builds. I checked with Maciej and he's OK with breaking it for now. The way it's currently implemented is wrong (you cannot throttle per task to simulate a limited bandwidth). We will need lower layer networking API/SPI to do it properly.